.github/workflows/ci-verification.yml

```diff
@@ -121,33 +102,23 @@ jobs:
     runs-on: [self-hosted, 1ES.Pool=gha-virtual-ccf-sub]
     container:
       image: ghcr.io/microsoft/ccf/ci/default:build-25-07-2024
+    defaults:
+      run:
+        working-directory: tla
 
     steps:
       - uses: actions/checkout@v4
-      - run: |
+      - name: Install TLC dependencies
+        run: |
           sudo apt update
           sudo apt install -y default-jre
-          python3 ./tla/install_deps.py
-
-      - name: MCabs.cfg
-        run: |
-          set -x
-          cd tla/
-          ./tlc.sh -workers auto consensus/MCabs.tla -dumpTrace tla MCabs.trace.tla -dumpTrace json MCabs.json
-
-      - name: MCccfraft - Configurations=1C2N
-        run: |
-          set -x
-          cd tla/
-          Configurations=1C2N MaxTermLimit=2 RequestLimit=3 ./tlc.sh -workers auto consensus/MCccfraft.tla -dumpTrace tla MCccfraft1C2NT2R3.trace.tla -dumpTrace json MCccfraft1C2NT2R3.json
+          python3 install_deps.py
 
-      - name: MCccfraft - Configurations=1C3N
-        run: |
-          set -x
-          cd tla/
-          Configurations=1C3N MaxTermLimit=2 RequestLimit=3 ./tlc.sh -workers auto consensus/MCccfraft.tla -dumpTrace tla MCccfraft1C3N2T3R.trace.tla -dumpTrace json MCccfraft1C3N2T3R.json
+      - run: ./tlc.py consensus/MCabs.tla
+      - run: ./tlc.py --max-term-count 2 --max-request-count 2 --raft-configs 1C2N --trace-name 1C2N consensus/MCccfraft.tla
```

Comment:
Restores previous behaviour with 2 term updates, and 2 requests. Limits with a strict comparison are misleading, RequestLimit=1 is really 0 requests, and MaxTermLimit=2 is really 0 term change, counts are hopefully clearer.